apps/dashboard/src/components/smart-wallets/index.tsx

```diff
@@ -19,23 +19,23 @@ export const SmartWallets: React.FC<SmartWalletsProps> = ({
   const searchParams = useSearchParams();
   const defaultTabIndex = Number.parseInt(searchParams?.get("tab") || "0");
   const [selectedTab, setSelectedTab] = useState<"factories" | "config">(
-    defaultTabIndex === 1 ? "config" : "factories",
+    defaultTabIndex === 1 ? "factories" : "config",
   );
 
   return (
     <div>
       <TabButtons
         tabs={[
           {
-            name: "Account Factories",
-            onClick: () => setSelectedTab("factories"),
-            isActive: selectedTab === "factories",
+            name: "Sponsorship Policies",
+            onClick: () => setSelectedTab("config"),
+            isActive: selectedTab === "config",
             isEnabled: true,
           },
           {
-            name: "Configuration",
-            onClick: () => setSelectedTab("config"),
-            isActive: selectedTab === "config",
+            name: "Account Factories",
+            onClick: () => setSelectedTab("factories"),
+            isActive: selectedTab === "factories",
             isEnabled: true,
           },
```

Comment:
since we're switching the order does the link from API Key settings still go to the right place?

Reply:
Good call, I changed the indexes back to their original pairings so it's a little weird now that the default index is 1 but all links should work just like before